backend/config/curation-api.yml

```diff
@@ -154,6 +154,8 @@ paths:
       description: >
         Fetch full metadata for the Collection and its Datasets.
 
+        If the collection owner's Authentication token is provided, then revision related fields will also be returned.
```

Comment:
I don't think we should use _collection owner's_ because we're discussing authorized users - for example, a super curator may not be the curator who created a collection, but they're authorized to see the details.

Reply:
```suggestion
        If an Authentication token is provided and the curator is allowed, then revision-related fields will also be returned.
```